pkg/container/device.go

```diff
@@ -163,3 +165,32 @@ func (c *container) specAddContainerConfigDevices() error {
 	}
 	return nil
 }
+
+func disableDeviceOwnershipFromSecurityContext(annotations map[string]string) bool {
+	if v, ok := annotations[crioann.DisableDeviceOwnershipFromSecurityContextAnnotation]; ok {
+		if v == "true" {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
we'll also need this annotation to be in gated by allowed annotations:
add a function in `internal/oci/oci.go` similar to `AllowUnifiedCgroupAnnotation` and pass that in to getDeviceUID

Reply:
done. does it look what you meant? I'll continue with more testing if this is what you had in mind.

Reply:
yes exactly, thanks!